src/main/java/kotlinx/reflect/lite/internal/ReflectProperties.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2010-2015 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kotlinx.reflect.lite.internal;
+
+import kotlin.jvm.functions.Function0;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.lang.ref.SoftReference;
+
+public class ReflectProperties {
```

Comment:
Why is https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/lazy.html not enough? Please leave a comment in the code.

Reply:
In any case, it doesn't look like this class should be public, right? If so, this is a good reminder to enable the explicit API mode.